lib/ruby_lsp/server.rb

```diff
@@ -38,10 +38,8 @@ module RubyLsp
             token_types: Requests::SemanticHighlighting::TOKEN_TYPES.keys,
             token_modifiers: Requests::SemanticHighlighting::TOKEN_MODIFIERS.keys,
           ),
-          range: false,
-          full: {
-            delta: true,
-          },
+          range: true,
+          full: { delta: false },
```

Comment:
Note: we never actually supported delta requests, so I'm not sure why this was `true`.